static/js/people.js

```diff
@@ -13,6 +13,13 @@ exports.get_by_email = function get_by_email(email) {
     return people_dict.get(email);
 };
 
+exports.get_star_status = function get_star_status(email) {
+    if (page_params.starred_users === undefined) {
+        return false;
+    }
+    return page_params.starred_users.indexOf(email)>=0;
```

Comment:
This ends up being quadratic (you call `get_star_status` on every user in the loop for `info_for`). Consider turning page_params.starred_users into an object and checking for existence of keys in the object rather than using array methods like `indexOf`